runtime/cudaq/spin/spin_op.cpp

```diff
@@ -6,18 +6,25 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  *******************************************************************************/
 
+#include "common/FmtCore.h"
 #include <cudaq/spin_op.h>
 #include <stdint.h>
 #ifdef CUDAQ_HAS_OPENMP
 #include <omp.h>
 #endif
 
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-anon-enum-enum-conversion"
+#endif
 #include <Eigen/Dense>
```

Comment:
Can we use EigenDense.h here and add this warning there?

Reply:
I think I simply missed replacing this, so yes.